### PR TITLE
Python-API: Support for 3D-coordinates with Maxent-Stress layout.

### PR DESCRIPTION
**Pull request by fabratu**
While the C++ code supports arbitrary dimensions, this is not easily exposed via Cython currently and require more work. This approach just adds 3D coordinates in addition to the already supported 2D coordinates. As a drive-by it fixes some wrong doc-strings. Complete doc-overhaul will be added as part of the current doc-improvement PR series.

### CODE REVIEW

File: networkit/viz.pyx [hunk 1]
```diff
@@ -60,21 +61,30 @@ cdef class GraphLayoutAlgorithm:
...
 		return self._this.numEdgeCrossings()
 
 	def getCoordinates(self):
-		""" Computes approximation (in parallel) of the Spanning Edge Centrality. """
+		""" Get node coordinates for computed layout."""
 		if self._this == NULL:
 			raise RuntimeError("Error, object not properly initialized")
-		cdef pair[double, double] pr = pair[double, double](0, 0)
+		cdef vector[double] pr3d = [0, 0, 0]
+		cdef vector[double] pr2d = [0, 0]
```
Comment (angriman) [inline networkit/viz.pyx:74]: Can't we create `pr3d` or `pr2d` depending on whether `num_dim` is 3 or 2 (by moving them inside the `if` statement below)?

**Review by angriman — DISMISSED** [review]

**Comment by fabratu** [comment]
Since the code is changed, I thought of including the documentation fixes for the viz-module in this PR. Or better put this on another later PR?

**Comment by angriman** [comment]
I think it's better to have the documentation fixes in a separate PR.

**Comment by fabratu** [comment]
Ok, removed the doc-fixes.

**Review by angriman — APPROVED** [review]

**Comment by fabratu** [comment]
Merging this earlier, since it is only a small fix and is needed for #891 .

**Pull request merged**